src/components/Sidebar.vue

```diff
@@ -27,61 +34,124 @@ export default {
   },
   methods: {
     showNewList(list) {
-      this.$emit('list', list.component);
+      this.$emit("list", list.component);
```

Comment:
don't really know what this does, but shouldn't this be a constant of some sort?